### ISSUE TITLE
Cannot install on Windows

### ISSUE THREAD

**Issue by RyenNelsen** [prrrnd/atom-git-projects#19]
Prior to v1.10.1, I was able to install without any issues on all machine types. Now when I attempt to either install fresh, or update, I get the following error:

```
> git-utils@3.0.0 install C:\Users\rnelsen\AppData\Local\Temp\apm-install-dir-115223-8984-dvdhgs\node_modules\git-projects\node_modules\git-utils
> node-gyp rebuild


C:\Users\rnelsen\AppData\Local\Temp\apm-install-dir-115223-8984-dvdhgs\node_modules\git-projects\node_modules\git-utils>node "C:\Users\rnelsen\AppData\Local\atom\app-0.187.0\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild 

gypnpm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\rnelsen\\AppData\\Local\\atom\\app-0.187.0\\resources\\app\\apm\\bin\\\\node.exe" "C:\\Users\\rnelsen\\AppData\\Local\\atom\\app-0.187.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\rnelsen\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\rnelsen\\.atom\\.apmrc" "install" "C:\\Users\\rnelsen\\AppData\\Local\\Temp\\d-115223-8984-18uywie\\package.tgz" "--target=0.21.0" "--arch=ia32"
npm ERR! node v0.10.35
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! git-utils@3.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the git-utils@3.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the git-utils package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls git-utils
npm ERR! There is likely additional logging output above.
```

I see that it could be git-utils, but the reason I am creating the error here is because I didn't have an issue prior to v1.10.1.

**Comment by prrrnd** [comment]
As you said, this seems to be an issue with git-utils for windows users. Some of them apparently found a fix/workaround. You can probably find more help in https://github.com/atom/git-utils/issues, [#28](https://github.com/atom/git-utils/issues/28) for instance.
I might consider adding a paragraph in the README regarding this. Thanks for reporting!

**Comment by RyenNelsen** [comment]
The main issue I had is that it worked before you updated git-projects. Nothing has changed on either of my systems, and I've never had Visual Studio installed on either of the systems, so that shouldn't be the fix.

**Comment by RyenNelsen** [comment]
In fact I just installed v1.10.1 on the one computer that never installed it in the first place, and it worked. So there is a problem with your software, not git-utils from what I can tell.

```
> apm install git-projects@1.10.1
Installing git-projects@1.10.1 to C:\Users\rnelsen\.atom\packages done
```

**Comment by prrrnd** [comment]
git-utils has been added in 1.11.0, so that explains why you can still install the 1.10.1.

**Comment by iyogeshjoshi** [comment]
I'm using Ubuntu and I'm also getting the same error while updating 'git-projects' I tried uninstalling and installing again, but same error again. [here](http://pastebin.com/FvFTwmup) is the debug log output.
P.S: I was trying to update from 1.9.0 -> 1.11.1

**Comment by prrrnd** [comment]
@RyenNelsen has created a new issue on the git-utils repo. https://github.com/atom/git-utils/issues/39

**Comment by RyenNelsen** [comment]
Alright, back again. @prrrnd atom/git-utils#39 is a bust. Any ideas? What are the exact requirements for your package?

**Comment by prrrnd** [comment]
What is the output of `npm install git-utils --msvs_version=2013` ?

**Comment by RyenNelsen** [comment]
Sorry in a different country right now, will try and get a response to this ASAP!

**Comment by vtintillier** [comment]
@prrrnd would it be possible to include a precompiled version of git-utils module?
It will be a headache if I need everybody in our teams to install Visual Studio just to install this Atom package :-(

**Comment by Efreak** [comment]
Agreed with @vtintillier. A precompiled version would make things so much simpler. I just spent the past hour+ trying to get this working. I have installed: atom, nodejs+npm via [nvm-windows](https://github.com/coreybutler/nvm-windows/releases), both VS 2013 Ultimate and 2015 Enterprise, python 2.7 (native) and 3.2 (cygwin), setuptools/pip, git/cygwin and github for windows. I'm running it under Windows 10 Pro x64.

It failed to install in the atom gui, so [I ran `apm install git-projects` in the command prompt](http://i.imgur.com/CTNMamB.png):
[![`apm install git-projects`](http://i.imgur.com/pBa4zqO.png)](http://i.imgur.com/pBa4zqO.png)
I looked around a bit online, including in the issues here and in the issues on the git-utils project, and found a single possible suggestion: Try installing git-utils with --msvs_version=2013. Since I have both 2013 and 2015 installed, I tried both ways.

[`npm install git-utils --msvs_version=2013`](http://i.imgur.com/WGhjSG8.png) and [`npm install git-utils --msvs_version=2015`](http://i.imgur.com/B3BYeDm.png) give the same result (so does 2012 tools, which are available through VS2013 but I didn't screenshot that):
[![npm install git-utils --msvs_version=2013](http://i.imgur.com/WGhjSG8.png)](http://i.imgur.com/SpvlCYM.png)

All of this would be much simpler if the binaries were already available. I *think* there's a way to include multi-platform binaries in a single package, but I'm not sure. If I can do anything to help get this working, please let me know.

**Comment by eigan** [comment]
I have the same issue on Mac OS 10.10.5. My coworker uses Linux and cannot upgrade/reinstall either.

**Comment by Bakudankun** [comment]
Is this issue already fixed?
I got same error before, but now I could install on Windows with no problem.

**Comment by Cr4ck3rs** [comment]
I did a lot of stuff to get this working, what finally did it was installing python 2.7 (with the path option checked)
Before that I had upgraded node-gyp running as admin from the node folder:
npm install -g node-gyp@latest
I also specified my vs version:
npm install -g node-inspector --msvs-version=2013
and also I installed the c++ redistributables for 2013 and 2010 and for both x86 and x64
